### PR TITLE
Added Power & Battery tab, added KaTeX support

### PR DESCRIPTION
**Pull request by VitroidFPV**
Was not exactly sure about some parts as I usually very rarely touch stuff here, please review and edit accordingly

### CODE REVIEW

**Review by SupaflyFPV — APPROVED** [review]

**Comment by haslinghuis** [comment]
image

**Comment by SupaflyFPV** [comment]
> image

ops I need to look those up...

**Review by blckmn — APPROVED** [review]

**Comment by KarateBrot** [comment]
You already included Katex? And already 100PRs... Damn, you guys are rolling 🚂

File: docs/configurator/power-battery-tab.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+sidebar-position: 4
+---
+
+# Power & Battery
+
+Set up all your power and battery related settings here. You can also see the current power measurements
+
+![Power & Battery tab](/img/betaflight_configurator_power_battery_tab.png)
```
Comment (haslinghuis) [inline docs/configurator/power-battery-tab.mdx:9]: image missing here

**Pull request merged**